### DISCUSSION TITLE
How to reuse docker format image storage from podman

### DISCUSSION THREAD

**Discussion by osadalakmal** [containers/podman#11986] [Q&A]
How can I reuse the docker format images I already have in ```/var/lib/docker``` with podman? I have multiple GBs worth of images that I will rather not download again. How can I get podman images command to look in ```/var/lib/docker``` instead of ```/var/lib/containers```? Do I have to go through the docker daemon unix socket? Or can I directly pass a file URI?

**Comment by rhatdan** [comment]
No, Podman and Docker use different storage mechanisms.
However you can pull the images from Docker directly into Podman 
podman pull docker-daemin:IMAGENAME

Podman talks to the Docker daemon, and pulls locally.  This will use twice the storage, but you will not need to pull over the network.
